### PR TITLE
replaces placeholder text and polishes preview page

### LINKED ISSUE #208: The Conscious Shopping text is disappearing with scroll on Safari
Screenshot 2020-11-08 at 13 46 03

### LINKED ISSUE #265: Add arrow with on click functionality on conscious shopping carousel

### LINKED ISSUE #287: Replace placeholder text in Conscious Shopping

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?
- added in the translated copy for each of the Conscious Shopping interviews
- made some tweaks on the project preview page to remove a weird scroll on desktop
- the carousel can now be moved using the arrow 

### Which issue(s) does this PR fix?
Fixes #287 
Fixes #208 
Fixes #265 

### Screenshots (if there are design changes)
Screenshot 2021-01-01 at 18 22 43

### How to test
- Move the date back in the Conscious Shopping page to be able to see the project preview page
- Check that the translation works on each of the different interviews and the arrow changes the interview on click
- Not picked up any issues on Safari, double-check on your side to make sure there are no sneaky display bugs

### CODE REVIEW

**Review by Jbarget — COMMENTED** [review]
It looks as if only the spanish versions have the "OX:" or "M:" for the question/answer text.
is that desired functionality?

apart from that all looks good

image

File: src/components/project-pages/ConsciousShoppingContent.tsx [hunk 1]
```diff
@@ -46,10 +51,29 @@ const SmallImage = styled.img<LayoutProps & FlexboxProps & SpaceProps>`
...
+  transition: transform 0.2s;
+  transform-origin: left;
+  &:hover {
+    transform: scale(1.05);
+  }
+`;
+
+const TextContainer = styled.div`
+  overflow-y: scroll;
+  ::-webkit-scrollbar {
```
Comment (Jbarget) [inline src/components/project-pages/ConsciousShoppingContent.tsx:69]: why did you need to do this?

"add scroll", "hide scroll bar"
seems like bad UI

File: src/components/project-pages/ConsciousShoppingPreview.tsx [hunk 2]
```diff
@@ -1,69 +1,79 @@
...
+import sunglasses from "../assets/conscious-shopping/sunglasses.jpg";
+import trex from "../assets/conscious-shopping/t-rex.jpg";
 import { zIndexes } from "../theme";
-import { Link } from "react-router-dom";
-import { PROJECTS_URL } from "../../constants/router-urls";
-
-type GridLayoutProps = FlexboxProps & LayoutProps & PositionProps & GridProps;
 
-const Grid = styled.div<GridLayoutProps>`
+const Div = styled.div<GridProps & LayoutProps & FlexboxProps>`
```
Comment (Jbarget) [inline src/components/project-pages/ConsciousShoppingPreview.tsx:25]: could this extend `Flex` component?

**Comment by sssoja** [comment]
yeee good q @Jbarget it's intended, not sure why. that's just how it's written in the document with all the copy

File: src/components/project-pages/ConsciousShoppingPreview.tsx (hunk 2 above)
Comment (sssoja) [reply src/components/project-pages/ConsciousShoppingPreview.tsx:25]: Not this time.. If you look on line 61, the div takes `flex` **and** `grid` as props which means that of we extended using `Flex` it would override the `grid` in the props array

File: src/components/project-pages/ConsciousShoppingContent.tsx (hunk 1 above)
Comment (sssoja) [reply src/components/project-pages/ConsciousShoppingContent.tsx:69]: Goood point

Comment (sssoja) [reply src/components/project-pages/ConsciousShoppingContent.tsx:69]: Basically what I was trying to do was hide the scrollbar while maintaining the scroll functionality. This should be fixed now.

**Comment by sssoja** [comment]
Going to sort the file structure out on #313 and once approved will merge into this PR

**Review by Jbarget — APPROVED** [review]
